ios/Classes/FirebaseDatabaseModule.m

```diff
@@ -16,6 +16,12 @@ @implementation FirebaseDatabaseModule
 
 #pragma mark Internal
 
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_VALUE, FIRDataEventTypeValue);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_ADD, FIRDataEventTypeChildAdded);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_REMOVE, FIRDataEventTypeChildRemoved);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_MOVE, FIRDataEventTypeChildMoved);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_CHANGE, FIRDataEventTypeChildChanged);
```

Comment:
Looks good! One typo I noticed (that was in my initial implementation as well) is that we should include the `CHILD` phrase as well, e.g.:
- `DATA_EVENT_TYPE_CHILD_ADDED`
- `DATA_EVENT_TYPE_CHILD_REMOVED`
- `DATA_EVENT_TYPE_CHILD_MOVED`
- `DATA_EVENT_TYPE_CHILD_CHANGED`

Can you change that in code and docs?